examples/job-catalog/src/openai.ts

```diff
@@ -74,6 +127,38 @@ client.defineJob({
       model: "text-embedding-ada-002",
       input: "The food was delicious and the waiter...",
     });
+
+    //Function call example for OpenAI
+    const createCompletion = async () => {
+      const chatCompletion = await io.openai.createChatCompletion("chat-completion", {
```

Comment:
I didn't understand this part @ericallam, 

Do I need to use await io.openai.chats.completions.create instead of await io.openai.createChatCompletion ?